enzyme/Enzyme/Enzyme.cpp

```diff
@@ -1081,6 +1111,10 @@ class Enzyme : public ModulePass {
       Optional<DIFFE_TYPE> opt_ty;
       Optional<StringRef> metaString = getMetadataName(res);
 
+      if (!metaString && isa<PHINode>(res)) {
```

Comment:
can you have this be part of getMetadataName?

Reply:
done